method/group.js

```diff
@@ -28,37 +35,39 @@ module.exports = function GroupMethods (ssb, keystore, state) {
       /* enveloping */
       // we have to do it manually this one time, because the auto-boxing checks for a known groupId
       // but the groupId is derived from the messageId of this message (which does not exist yet
-      const plain = Buffer.from(JSON.stringify(content), 'utf8')
+      //const plain = Buffer.from(JSON.stringify(content), 'utf8')
 
-      const msgKey = new SecretKey().toBuffer()
-      const recipientKeys = [
+      //const msgKey = new SecretKey().toBuffer()
+      const recps = [
         { key: groupKey.toBuffer(), scheme: keySchemes.private_group },
-        keystore.self.get() // sneak this in so can decrypt it ourselves without rebuild!
+        ssb.id // sneak this in so can decrypt it ourselves without rebuild!
```

Comment:
does this work ok? it might, but then it presumes recps can take a jumble of actual keys and feedIds which get mapped to keys.... tests will tell you if it doesn't work

Reply:
yeah i swapped to this because tests were failing, this works better